content/docs/1.13/apis.md

```diff
@@ -0,0 +1,81 @@
+---
+title: APIs
+hasparent: true
+---
+
+Jaeger components implement various APIs for saving or retrieving trace data.
+
+TODO
+
+  * stable
+  * restricted
+  * deprecated
+
+## Span reporting APIs
+
+Agent and Collector are the two components of the Jaeger backend that can receive spans and forward them to storage.
```

Comment:
This might be confusing, agent cannot forward data to storage

Reply:
they both do, indirectly (e.g collector may also not talk to storage when Kafka is used).  But I can rephrase.